diff -Nur -x '*.orig' -x '*~' terminator-0.10/terminatorlib/configfile.py terminator-0.10.new/terminatorlib/configfile.py
--- terminator-0.10/terminatorlib/configfile.py	2008-08-28 09:17:49.000000000 -0500
+++ terminator-0.10.new/terminatorlib/configfile.py	2008-09-06 15:07:54.000000000 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 
 import re
 from terminatorlib.config import dbg, debug
diff -Nur -x '*.orig' -x '*~' terminator-0.10/terminatorlib/debugserver.py terminator-0.10.new/terminatorlib/debugserver.py
--- terminator-0.10/terminatorlib/debugserver.py	2008-08-28 09:17:49.000000000 -0500
+++ terminator-0.10.new/terminatorlib/debugserver.py	2008-09-06 15:07:34.000000000 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 #
 # Copyright (c) 2008, Thomas Hurst <tom@hur.st>
 #
diff -Nur -x '*.orig' -x '*~' terminator-0.10/terminatorlib/freebsd.py terminator-0.10.new/terminatorlib/freebsd.py
--- terminator-0.10/terminatorlib/freebsd.py	2008-08-28 09:17:49.000000000 -0500
+++ terminator-0.10.new/terminatorlib/freebsd.py	2008-09-06 15:07:45.000000000 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 #
 # Copyright (c) 2008, Thomas Hurst <tom@hur.st>
 #
